app/__init__.py

```diff
@@ -1,13 +1,25 @@
 import os
+import sentry_sdk
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
 from flask_moment import Moment
 from flask_login import LoginManager
 from flask_misaka import Misaka
+from sentry_sdk.integrations.flask import FlaskIntegration
 
+sentry_dsn = os.environ.get("SENTRY_DSN")
+sentry_id = os.environ.get("SENTRY_ID")

```

Comment:
What happened if users don’t add sentry config? Will this break the app? If yes, maybe should add logic where unavailability of this config disable the initiation of sentry sdk.

Reply:
if users do not add sentry env_vars it just not working, same as the other env_vars in config object